instapy/login_util.py

```diff
@@ -253,7 +253,7 @@ def login_user(browser,
         update_activity()
 
     login_button = browser.find_element_by_xpath(
-        "//button[text()='Log in']")
+        "//button[contains(.,'Log in')]")
```

Comment:
```suggestion
        "//button[contains(.,'Log in')]")
```

What does the `.` do before the comma? Is it referring to the currently selected element?

Reply:
https://stackoverflow.com/questions/46486477/why-wont-my-xpath-select-link-button-based-on-its-label-text

Hope this helps

Reply:
https://stackoverflow.com/questions/38240763/xpath-difference-between-dot-and-text

This one has better examples